src/util/keys.ts

```diff
@@ -40,6 +40,3 @@ export const RecordingsFolderKey = '${RECORDINGS}/';
 export const WindowsKey = 'Windows';
 export const ScreenSpaceKey = 'ScreenSpace';
 export const RenderableKey = 'Renderable';
-export const ScaleKey = 'Scale';
```

Comment:
OBS Leaning in to a reduced use of keys. It does make the code easier to read, since this is how we've started doing it more and more